features/attached_enable.feature

```diff
@@ -1160,7 +1159,6 @@ Feature: Enable command behaviour when attached to an Ubuntu Pro subscription
         """
 
     @series.xenial
-    @uses.config.contract_token
```

Comment:
Same question here, as we have an attach operation as one of the steps

Reply:
same answer